taxi/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = ("model", "manufacturer", "drivers",)
+
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The `queryset` for the `drivers` field in `CarForm` should use `Driver.objects.all()` instead of `get_user_model().objects.all()`. This ensures that the queryset is correctly fetching `Driver` instances, which is necessary for assigning drivers to cars.